### PR TITLE
[ITT-477] Adjust README in preparation of release

### PR DESCRIPTION
**Pull request by jillingk**
**Description**
This PR updates the README similarly to what we have done in the previous libraries.

### CODE REVIEW

**Comment by jillingk** [comment]
Also added a makefile line to revert to camelCase naming for easy arrayAccess. Will create a follow up PR to regenerate and adjust the tests.

File: Makefile
Comment (wboereboom) [inline Makefile:1]: Why are there Makefile changes in the README pr?

File: README.md
Comment (wboereboom) [inline README.md:1]: Readme looks good!

File: Makefile
Comment (jillingk) [reply Makefile:1]: woops

**Review by antolo-arch — APPROVED** [review]

**Review by wboereboom — APPROVED** [review]

**Pull request merged**